library/system/src/lib/yast2/system_service.rb

```diff
@@ -56,6 +56,16 @@ class SystemService
     #   underlying system.
     attr_reader :errors
 
+    # @!method state
+    #
+    # @return [String]
+    def_delegator :@service, :active_state, :state
+
+    # @!method substate
+    #
+    # @return [String]
+    def_delegator :@service, :sub_state, :substate
+
     def_delegators :@service, :running?, :start, :stop, :restart, :active?,
```

Comment:
NP: I would not forward `#active_state` and `sub_state` directly.  It is true that right now we are simply forwarding that methods with a different name, but those systemd values should be "internal" details.